utils/frame/try-runtime/remote-externalities/Cargo.toml

```diff
@@ -13,16 +13,13 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-jsonrpc-core-client = { version = "15.1.0", features = ["http"] }
-sc-rpc-api = { version = "0.9.0", path = "../../../../client/rpc-api" }
-sc-rpc = { version = "3.0.0", path = "../../../../client/rpc" }
-futures = "0.1.29"
+jsonrpsee-http-client = { git = "https://github.com/paritytech/jsonrpsee", default-features = false, features = ["hyper-client"] }
```

Comment:
this works with `tokio 0.2` and `async std`

Reply:
can you pin to a rev?

Reply:
also, @gnunicorn I recall that you had some grudge against using git deps, and that we had to get rid of them prior to releases? Is this okay, given that the code is only used in a niche testing cli utility.

Reply:
the problem is that crates.io doesn't allow for git-dependencies and as this is _not_ marked `publish = false`, this means this crate will fail when we try to publish.

Reply:
right, I think we could release alpha/beta release for the JSONRPC clients only.

//cc @dvdplm @maciejhirsz

Reply:
yeah, let's aim for having a release of jsonrpsee as well while we're at it.